reference/promise-types/measurements.markdown

```diff
@@ -20,27 +20,108 @@ bundle destined for the agent concerned, in this case `monitor`. However, it is
 not necessary to add them to the `bundlesequence`, because `cf-monitord`
 executes all bundles of type `monitor`.
 
+Information gathered by `cf-monitord` is stored in `${sys.statedir}/env_data` to
+share with other agents like `cf-agent` and `cf-promises` which both provide three
+variables per measurement with different prefixes: `value_` for the latest measurement,
+`av_` for the running average and `dev_` for the standard deviation of the running average.
+
+The standard measurements and their variable names are listed here: https://docs.cfengine.com/docs/master/reference-special-variables-mon.html
+
+The data over time is stored in `${sys.statedir}/cf_observations.lmdb` which can be examined
```

Comment:
Let's review the description here too:
https://docs.cfengine.com/docs/3.18/guide-introduction-directory-structure.html

```
state/cf_observations.lmdb
This database contains the current state of the observational history of the host as recorded by cf-monitord.
```